Assets/Code/Scripts/PlayerManagement/RealityPlayerCollisions.cs

```diff
@@ -27,7 +27,8 @@ private void Awake()
         }
         else if (other.CompareTag("OutOfBounds"))
         {
-            _respawningManager.RespawnAllTransforms();
+            //_respawningManager.RespawnAllTransforms();
+            EventManager.TriggerEvent("FadeOutRespawn");
             EventManager.TriggerEvent("DisplayHint", "falling down hurts... (right click to skip animation)");
```

Comment:
```suggestion
            //_respawningManager.RespawnAllTransforms();
            EventManager.TriggerEvent("FadeOutRespawn");
```

We can remove the hint